make/Docs.gmk

```diff
@@ -37,7 +41,7 @@ $(eval $(call ReadImportMetaData))
 
 ################################################################################
 # Hook to include the corresponding custom file, if present.
-$(eval $(call IncludeCustomExtension, Docs.gmk))
+#$(eval $(call IncludeCustomExtension, Docs.gmk))
```

Comment:
It's disappointing that this file needs to change. Did you try alternative approaches in `close/make/Docs.gmk`?

Reply:
I tried very hard to get this to work by using the plugin point provided already.  Unfortunately some macros which need to be modified are changed after the include point, so they are overwritten.  Moving the include down a few lines works, but I'm not exactly comfortable with the 'solution'.  Perhaps it would be better to leave the existing include in place and an an extra include for a new file (e.g. Docs-post.gmk) in the new position.
If anyone can see how to make it work without this new include point I'd be very happy.

Reply:
I'm happy to give it a try once you get past the build problem I noted. I'm hoping it just requires careful use of `=` instead of `:=` so the timing is right for macro expansion.

Reply:
I've moved the openj9 javadoc generation code to a new Docs-post.gmk file so as not to move the original closed/make/Docs.gmk include in make/Docs.gmk, and added a new include for closed/make/Docs-post.gmk.  This should at least reduce the chance of future merge conflicts.